src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,147 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+
+import duke.command.AddCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Todo;
+
+/**
+ * Represents a parser
+ */
+public class Parser {
+
+    private static final String EXIT_COMMAND = "bye";
```

Comment:
Perhaps shifting the "COMMAND" suffix to be a prefix would be more in line with the coding standard.